src/patterns/data/default/entity_attribute.tsv

```diff
@@ -4271,116 +4271,4 @@ OBA:2045222	basophilic erythroblast count	CL:0000549	basophilic erythroblast	PAT
-OBA:2045419		UBERON:0001359	cerebrospinal fluid	PATO:0000014	color			https://orcid.org/0000-0001-8314-2140
-OBA:2045420		UBERON:0001417	skin of neck	PATO:0000915	thickness			https://orcid.org/0000-0001-8314-2140
-OBA:2045421	semen attribute	UBERON:0001968	semen	PATO:0000001	quality			https://orcid.org/0000-0001-8314-2140
-OBA:2045422		UBERON:0001968	semen	PATO:0000918	volume			https://orcid.org/0000-0001-8314-2140
-OBA:2045423		UBERON:0001968	semen	PATO:0000992	viscosity			https://orcid.org/0000-0001-8314-2140
-OBA:2045424	semen pH	UBERON:0001968	semen	PATO:0001842	acidity			https://orcid.org/0000-0001-8314-2140
-OBA:2045425	plasma pH	UBERON:0001969	blood plasma	PATO:0001842	acidity			https://orcid.org/0000-0001-8314-2140
-OBA:2045426		UBERON:0001969	blood plasma	PATO:0002027	osmolality			https://orcid.org/0000-0001-8314-2140
-OBA:2045427		UBERON:0001977	blood serum	PATO:0000992	viscosity			https://orcid.org/0000-0001-8314-2140
-OBA:2045428		UBERON:0001988	feces	PATO:0000037	consistency	firmness of faeces		https://orcid.org/0000-0001-8314-2140
-OBA:2045429		UBERON:0001988	feces	PATO:0000125	mass	faeces mass		https://orcid.org/0000-0001-8314-2140
-OBA:2045430	feces pH	UBERON:0001988	feces	PATO:0001842	acidity	pH of faeces		https://orcid.org/0000-0001-8314-2140
-OBA:2045431	body fluid attribute	UBERON:0006314	bodily fluid	PATO:0000001	quality			https://orcid.org/0000-0001-8314-2140
-OBA:2045432	body fluid color	UBERON:0006314	bodily fluid	PATO:0000014	color			https://orcid.org/0000-0001-8314-2140
-OBA:2045433	body fluid pH	UBERON:0006314	bodily fluid	PATO:0001842	acidity			https://orcid.org/0000-0001-8314-2140
-OBA:2045434	umbilical cord blood pH	UBERON:0012168	umbilical cord blood	PATO:0001842	acidity			https://orcid.org/0000-0001-8314-2140
-OBA:2045435	stomach content pH	UBERON:0013525	stomach lumen	PATO:0001842	acidity			https://orcid.org/0000-0001-8314-2140
-OBA:2045436	arterial blood pH	UBERON:0013755	arterial blood	PATO:0001842	acidity			https://orcid.org/0000-0001-8314-2140
-OBA:2045437	venous blood pH	UBERON:0013756	venous blood	PATO:0001842	acidity			https://orcid.org/0000-0001-8314-2140
-OBA:2045438	capillary blood pH	UBERON:0013757	capillary blood	PATO:0001842	acidity			https://orcid.org/0000-0001-8314-2140
-OBA:2045439	body surface area	UBERON:0035159	entire surface of organism	PATO:0001323	area			https://orcid.org/0000-0001-8314-2140
-OBA:2045451	attribute of response to  allergen in serum	UBERON:0001977	blood serum	OBA:2045453	attribute of response to allergen			https://orcid.org/0000-0001-8314-2140
-OBA:2045452	attribute of response to thyroid hormone in plasma	UBERON:0001969	blood plasma	OBA:2045454	attribute of response to thyroid hormone			https://orcid.org/0000-0001-8314-2140
-OBA:2030004	total skin mass	UBERON:0002097	skin of body	PATO:0000125	mass	mass of the entire skin		https://orcid.org/0000-0003-2699-3066
```

Comment:
Please, do not remove OBA:2045226 through OBA:2045226.
You need to update your `entity_attribute.tsv` file to the latest version from the `master` branch before you add OBA:2050351.